apps/client-ui/scripts/deploy-amplify-preview.sh

```
@@ -0,0 +1,366 @@
+    if [[ "$JOB_INFO" != "{}" ]]; then
+        JOB_STATUS=$(echo "$JOB_INFO" | jq -r '.job.summary.status' 2>/dev/null || echo "UNKNOWN")
+        JOB_START_TIME=$(echo "$JOB_INFO" | jq -r '.job.summary.startTime' 2>/dev/null || echo "")
+
+        # Only print status changes to reduce noise
+        if [[ "$JOB_STATUS" != "$LAST_STATUS" ]]; then
+            case "$JOB_STATUS" in
+                "SUCCEED")
+                    print_status "Deployment completed successfully"
+
+                    # Get deployment URL and additional info
+                    DEPLOYMENT_URL=$(echo "$JOB_INFO" | jq -r '.job.summary.jobArn' 2>/dev/null || echo "")
```

Shellcheck (0.10.0)

[warning] DEPLOYMENT_URL appears unused. Verify use (or export if used externally).

(SC2034)